src/QueryBuilder.php

```diff
@@ -69,24 +69,8 @@ public function getSearchParams(string $queryString, string $type, int $start, i
         $queryString = str_replace(['"', '\''], '', $queryString);
         $queryString = htmlspecialchars_decode($queryString);
         $blocks = preg_split( '/( AND )/i', $queryString);
```

Comment:
Space after `(`

Reply:
I did not touch this part. In order to prevail PR excessive size, I avoid any code reformatting that is not directly related to new code that I implemented.